### DISCUSSION TITLE
Easy responsive

### DISCUSSION THREAD

**Discussion by Adam-maras** [tailwindlabs/tailwindcss#11224] [Ideas]
Instead of md:text-lg md:text-red-600

It would be great to have 

md:(text-lg text-red-600)

**Comment by wongjn** [comment]
There have been many suggestions about adding such a feature, but [here's one such response from a maintainer](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

[Also another thread here](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611) with more responses from maintainers.
